ghost/i18n/locales/dv/portal.json

```diff
@@ -0,0 +1,216 @@
+    "Success! Your email is updated.": "ކާމިޔާބު! ތިބާގެ އީމެއިލް އަޕްޑޭޓް ކުރެވިއްޖެ.",
+    "Successfully unsubscribed": "ކާމިޔާބުކަމާއެކު އަންސަބްސްކްރައިބް ކުރެވިއްޖެ",
+    "Thank you for subscribing. Before you start reading, below are a few other sites you may enjoy.": "ސަބްސްކްރައިބް ކުރެއްވީތީ ޝުކުރިއްޔާ. ކިޔަން ފެށުމުގެ ކުރިން، ތިރީގައި މިއޮތީ ތިޔަބޭފުޅާއަށް ކަމުދާނެ އެހެން ސައިޓްތަކެކެވެ.",
+    "Thank you for your support": "ތިބާގެ ސަޕޯޓަށް ޝުކުރިއްޔާ",
+    "Thank you for your support!": "ތިބާގެ ސަޕޯޓަށް ޝުކުރިއްޔާ!",
+    "Thanks for the feedback!": "ފީޑްބެކް ދެއްވީތީ ޝުކުރިއްޔާ!",
+    "That didn't go to plan": "އެކަން ޕްލޭނާއި އެއްގޮތަށް ނުދިޔަ",
+    "The email address we have for you is {{memberEmail}} — if that's not correct, you can update it in your <button>account settings area</button>.": "އަހަރެމެންގެ އަތުގައި ތިބާގެ އީމެއިލް އެޑްރެސްއަކީ {{memberEmail}} — އެއީ ރަނގަޅު ނޫންނަމަ، ތިބާގެ <button>އެކައުންޓް ސެޓިންގްސް އޭރިއާ</button> އިން އަޕްޑޭޓް ކުރެވިދާނެއެވެ.",
+    "There was a problem submitting your feedback. Please try again a little later.": "ތިބާގެ ފީޑްބެކް ސަބްމިޓްކުރުމުގައި މައްސަލައެއް ދިމާވެއްޖެ. ފަހުން އަލުން ޓްރައި ކުރައްވާ.",
+    "There was an error cancelling your subscription, please try again.": "ތިބާގެ ސަބްސްކްރިޕްޝަން ކެންސަލްކުރުމުގައި އެރަރެއް ދިމާވެއްޖެ، އަލުން ޓްރައި ކުރައްވާ.",
+    "There was an error continuing your subscription, please try again.": "ތިބާގެ ސަބްސްކްރިޕްޝަން ކުރިއަށްގެންދިއުމުގައި އެރަރެއް ދިމާވެއްޖެ، އަލުން ޓްރައި ކުރައްވާ.",
+    "There was an error processing your payment. Please try again.": "ތިބާގެ ޕޭމަންޓް ޕްރޮސެސްކުރުމުގައި އެރަރެއް ދިމާވެއްޖެ. އަލުން ޓްރައި ކުރައްވާ.",
+    "There was an error sending the email, please try again": "އީމެއިލް ފޮނުވުމުގައި އެރަރެއް ދިމާވެއްޖެ، އަލުން ޓްރައި ކުރައްވާ",
+    "This site is invite-only, contact the owner for access.": "މި ސައިޓްއަކީ އިންވައިޓް-އޮންލީ ސައިޓެއް، އެކްސެސް ހޯދުމަށް ވެރިފަރާތާއި ގުޅުއްވާ.",
+    "This site is not accepting payments at the moment.": "މި ސައިޓުން މިވަގުތު ޕޭމަންޓް ނުނަގާ.",
+    "This site only accepts paid members.": "މި ސައިޓުން ހަމައެކަނި ޕެއިޑް މެމްބަރުން އެކަނި ބަލައިގަނޭ.",
+    "To complete signup, click the confirmation link in your inbox. If it doesn't arrive within 3 minutes, check your spam folder!": "ސައިންއަޕް ފުރިހަމަކުރުމަށް، ތިބާގެ އިންބޮކްސްއިން ކޮންފޮރމޭޝަން ލިންކަށް ކްލިކްކުރައްވާ. 3 މިނެޓްގެ ތެރޭގައި ނުލިބިއްޖެނަމަ، ތިބާގެ ސްޕޭމް ފޯލްޑަރ ޗެކްކުރައްވާ!",
+    "To continue to stay up to date, subscribe to {{publication}} below.": "އަޕް ޓު ޑޭޓްކޮށް ހުރުމަށް، ތިރީގައިވާ {{publication}} އަށް ސަބްސްކްރައިބްކުރައްވާ.",
+    "Too many attempts try again in {{number}} days.": "މާ ގިނަ ފަހަރު މަސައްކަތްކޮށްފއިވޭ، {{އަދަދު}} ދުވަހުން އަލުން ޓްރައި ކުރައްވާ.",
+    "Too many attempts try again in {{number}} hours.": "މާ ގިނަ ފަހަރު މަސައްކަތްކޮށްފއިވޭ، {{އަދަދު}} ގަޑިއިރުން އަލުން ޓްރައި ކުރައްވާ.",
```

Comment:
there are some more translations of variables here that need to be corrected or the translations will not work.